src/pages/legality.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from "react";
+import { Helmet } from "react-helmet";
+import { LegalityPageContent } from "../components/LegalityPageContent";
+
+export const LegalityPage: FunctionComponent = () => (
+  <>
+    <Helmet>
+      <meta name="description" content="A framework that was designed to achieve legal interoperability." />
+      <meta property="og:description" content="A framework that was designed to achieve legal interoperability." />
+      <meta property="og:title" content="TradeTrust - Legality" />
+      <meta property="og:url" content={`${window.location.origin}/eta`} />
+      <title>TradeTrust - Legality</title>
+      <meta
+        name="keywords"
+        content="Blockchain, NFT, Ethereum, Electronic Trade Document, Verifiable Document, Digital Trade Document, Transferable Documents, Electronic Transactions Act, Bill of Lading"
```

Comment:
Do we have a parent component to house these common meta tags for all pages?

Reply:
if so i think i can refactor the meta data to [here](https://github.com/TradeTrust/tradetrust-website/blob/master/src/routes.tsx#L52-L74), then have a common "head" component. Or do you any other idea in mind?

Reply:
Just thought in case we have a parent component and missed it in this PR only, don’t think it’s necessary to implement it into this PR if we don’t already have one.